Eastern-Research-Group/mywaterway
# Feature/545 expand on other uses

## Related Issues:
* [HMW-545](https://jira.epa.gov/browse/HMW-545)

## Main Changes:
* Switched from using `@reach/dialog` to `@radix-ui/react-dialog`
* Consolidated the modal components
* Added buttons to each use in the waterbody content that displays the detailed uses

## Steps To Test:
1. Navigate http://localhost:3000/community/dc/overview
2. Click the `Disclaimer` button
3. Verify the disclaimer opens and looks correct
4. Open one of the waterbodies
5. Click the ? icons
6. Verify they display detailed uses and that those detailed uses make sense for the use you clicked
7. Be sure to do steps 4 - 6 for both the list view and the popup view
8. Navigate to http://localhost:3000/waterbody-report/DOEE/DCANA00E_02/2020
9. Repeat steps 4 - 6 for the map popup
10. Navigate to http://localhost:3000/state/AL/advanced-search
11. Perform a few searches and verify the confirm modal works correctly
